app/models/hyrax/batch_ingest/ability.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module Hyrax
+  module BatchIngest
+    module Ability
+      extend ActiveSupport::Concern
+      included do
+        self.ability_logic += [:admin_abilities]
+      end
+
+      def admin_abilities
+        return unless admin?
+        can [:create, :show, :index, :read, :edit, :update, :destroy], Hyrax::BatchIngest::Batch
```

Comment:
If an admin can do anything then how about making this `can :manage, Hyrax::BatchIngest::Batch`?

Reply:
Yes. I think that's a great idea for later. For now, I like that it shows how to enumerate individual perms.